Formula/pygobject3.rb

```diff
@@ -16,22 +15,33 @@ class Pygobject3 < Formula
   option "without-python", "Build without python3 support"
   option "with-python@2", "Build with python2 support"
 
+  depends_on "meson" => :build
+  depends_on "pkg-config" => :build
```

Comment:
These dependencies should be listed in alphabetical order. `pkg-config` is also listed twice.